Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "gleam"
 version = "0.13.2"
 authors = ["Louis Pilfold <louis@lpil.uk>"]
 edition = "2018"
+license-file = "LICENSE"
```

Comment:
Can we use LICENCE rather than LICENSE? I'm not American :)

Reply:
Sure! (I am, so I presumed it to be a typo. I actually just used the spelling of cargo-deny, was always unsure before now!).